app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -23,6 +23,7 @@ import AcuantContext, { AcuantCaptureMode } from '../context/acuant';
 import AnalyticsContext from '../context/analytics';
```

Comment:
1. I forgot to add them to the analytics 😅 [fixed](https://github.com/18F/identity-idp/pull/10156/commits/bd90f48e788682a35db2bb16730f3770ce63a931).
2. 🤷 Seems reasonable to me. I'm comfortable merging with just automated tests for these.
3. Those are coming for free since I added to the `getAddAttemptAnalyticsPayload`. I think it's good to have the new property more places.

Reply:
Thanks for fixing #1! The rest makes sense to me.